src/resourcemanager/resourcemanager.cpp

```diff
@@ -71,6 +345,16 @@ Error JSONProvider::ParseNodeConfig(const String& json, sm::resourcemanager::Nod
 
     auto parsedNodeConfig = MakeUnique<NodeConfig>(&mAllocator);
 
+    memset(parsedNodeConfig.Get(), 0, sizeof(NodeConfig));
```

Comment:
```parsedNodeConfig = {};```

Reply:
Looks like we don't need this memset at all
All the fields are explicitly initialized below.

BTW, parsedNodeConfig has all it fields set to zero after MakeUnique. (checked with gdb)